cra-config.yaml

```diff
@@ -8,3 +8,4 @@ CRA_TARGETS:
       TF_VAR_hpcs_key_crn_worker_pool: "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9:key:1368d2eb-3ed0-4a8b-b09c-2155895f01ea"
       TF_VAR_region: "us-south"
       TF_VAR_prefix: "base-ocp-std"
+      TF_VAR_ocp_version: "4.15"
```

Comment:
I would not add this here - it will get out of date. The code is set up to pick the current IKS default version, which is currently 4.15. So not sure why you added this here?

Reply:
This was added as the CRA scan step was [failing](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/actions/runs/9838973718/job/27160205109) in pipeline, by providing `ocp_version`, it passes the CRA scan failure step.

Reply:
@imprateeksh then you have a bug in your code. Passing no value for `ocp_version` is supported. When you do this, the code will do a data lookup to determine the current default OCP version

CRA is failing with this this:
```
│ Error: Invalid function argument
│ 
│   on ../../main.tf line 20, in locals:
│   20:   validate_ocp_version_for_rhcos = local.rhcos_validation ? true : tobool("RHCOS requires VPC clusters created from 4.15 onwards. Upgraded clusters from 4.14 cannot use RHCOS.")
│     ├────────────────
│     │ while calling tobool(v)
│ 
│ Invalid value for "v" parameter: cannot convert "RHCOS requires VPC
│ clusters created from 4.15 onwards. Upgraded clusters from 4.14 cannot use
│ RHCOS." to bool; only the strings "true" or "false" are allowed.
```

However the default version is 4.15, so this should not fail. Double check your logic here

Reply:
Noted. I understood your point. I have removed this entry from `cra-config.yaml`